### DISCUSSION TITLE
transient not reading values file when location is custom

### DISCUSSION THREAD

**Discussion by mtwomey** [magit/transient#200] [Support] [answered]
### What did you expect to happen?

I update my transient variables:

```
  (setq transient-values-file "~/.doom.d/.transient/values.el"
       transient-history-file "~/.doom.d/.transient/history.el"
       transient-levels-file "~/.doom.d/.transient/levels.el"))
```

Now the saved transient preferences in magit are saved and loaded from these files.

### What actually happened?

This works for _writing_. Saving preferences in magit (with C-x C-s) saves values to this location (in the values file). However after restarting emacs the values are not read in magit. Note: the variables are still successfully set to this custom location, the file is simply not being used.

### Describe your attempts to resolve the issue

Tried setting these variables at different times, was unable to find a setup that worked.

### Steps to reproduce

1. Set the variables to point to a different custom location
1. Save some new preferences
1. Quite and restart emacs
1. Observe if the preferences are taking effect in magit

**Comment by tarsius** [comment, marked as answer]
These variables are used when `transient` is loaded, so if you set them after it has already been loaded, then that has no effect.

**Comment by mtwomey** [comment]
On thanks, confirmed this is related to doom package loading details.
